subset/bacnet/bacnetTests/src/main/java/FauxDeviceEngine/Analog.java

```diff
@@ -20,16 +20,16 @@
     private static float deadband = 14;
     private static boolean outOfService = false;
     private static float resolution = 0.1f;
-    private static boolean[] eventEnable = {true, true, false};
+    private static boolean[] eventEnable = new boolean[3];
```

Comment:
where does the initialization happen now?  Also, these are static variables so it's shared class state -- are you sure that's the right thing?